src/ln/channelmonitor.rs

```diff
@@ -748,7 +748,7 @@ impl ChannelMonitor {
 	fn block_connected(&self, txn_matched: &[&Transaction], height: u32, broadcaster: &BroadcasterInterface) {
 		for tx in txn_matched {
 			for txin in tx.input.iter() {
-				if self.funding_txo.is_none() || (txin.prev_hash == self.funding_txo.unwrap().0 && txin.prev_index == self.funding_txo.unwrap().1 as u32) {
+				if self.funding_txo.is_none() || (txin.prev_hash == self.funding_txo.unwrap().txid && txin.prev_index == self.funding_txo.unwrap().index as u32) {
```

Comment:
Please make a note somewhere (maybe just file an issue when this is merged?) that whoever writes the serializer/deserializer for ChannelMonitor must use u32s or u16s for indexes (though I'm kinda not a fan of either, given the usize is platform-dependant).